administrator/components/com_content/src/Model/FeaturedModel.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_content
+ *
+ * @copyright   (C) 2009 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Content\Administrator\Model;
+
+use Joomla\CMS\MVC\Factory\MVCFactoryInterface;
+use Joomla\Database\QueryInterface;
```

Comment:
```suggestion
namespace Joomla\Component\Content\Administrator\Model;
```
Remove unused `use` statements so PHPCS will (hopefully) finally be happy.